src/output-json-alert.c

```diff
@@ -542,6 +543,12 @@ static void AlertAddAppLayer(const Packet *p, JsonBuilder *jb,
         case ALPROTO_RDP:
             AlertJsonRDP(p->flow, tx_id, jb);
             break;
+        case ALPROTO_MODBUS:
```

Comment:
Is this an addition ? (and not just the translation of the C parser)
If so, it deserves a specific commit

Reply:
Yes, it is an addition and actually resolves a separate redmine ticket which I noticed after we implemented this: [2096](https://redmine.openinfosecfoundation.org/issues/2096) as opposed to [3957](https://redmine.openinfosecfoundation.org/issues/3957).